### PR TITLE
add note about `winget` to README (for windows installations)

### PR DESCRIPTION
**Pull request by rursprung**
[winget] is the official command-line package manager for Windows starting with Windows 10, thus there's no need to use 3rd party package managers anymore.
both `avr-gcc` and `avrdude` are packaged in winget.

installing them this way works fine and no further manual modifications of the environment are needed (tested in Git Bash and Powershell).

the note about scoop has been left in the README as i wasn't sure whether there were still any `var-hal` users left on older Windows releases, though at some point it can/should be removed for being obsolete.

[winget]: https://learn.microsoft.com/en-us/windows/package-manager/winget/

### CODE REVIEW

**Comment by rursprung** [comment]
i've also updated [the wiki page](https://github.com/Rahix/avr-hal/wiki/Setting-up-environment) accordingly.

**Comment by stappersg** [comment]
On Wed, Dec 13, 2023 at 01:05:33AM -0800, Ralph Ursprung wrote:
> [winget] is the official command-line package manager for Windows
> starting with Windows 10, thus there's no need to use 3rd party package
> managers anymore.

That statement contradicts the provided link.


Regards
Geert Stappers

[winget]: https://learn.microsoft.com/en-us/windows/package-manager/winget/
-- 
Silence is hard to parse

**Comment by rursprung** [comment]
> That statement contradicts the provided link.

huh, in what way?

**Comment by stappersg** [comment]
On Thu, Dec 14, 2023 at 01:43:42AM -0800, Ralph Ursprung wrote:
> > That statement contradicts the provided link.
> 
> huh, in what way?

The statement: All fine starting with Windows 10

The provided link: "part of the App Installer. You can get App Installer from the Microsoft Store"
and "only supported on Windows 10 1709 (build 16299) or later"


Other problem I have with current proposed merge request
is that it claims position above existing solution and
(dis)qualifing that solution, `scoop`, to older systems.


Suggestion for next version of "Winget is the best!"

* Below   scoop.sh
* Windows 10 1709 (build 16299) or later
* "get App Installer from the Microsoft Store"
* Now you should have "winget"
* `winget install AVRDudes.AVRDUDE ZakKemble.avr-gcc`

 
Groeten
Geert Stappers
-- 
Silence is hard to parse

**Review by Rahix — APPROVED** [review]
I have no stakes in the windows package management world but I think this addition is entirely reasonable.  Thanks a lot for the contribution!
